src/caf.js

```diff
@@ -28,7 +28,22 @@ module.exports.signalAll = signalAll;
 
 
 function CAF(generatorFn) {
-	return function instance(tokenOrSignal,...args){
+	// allow users to introspect to understand if the function they're dealing
```

Comment:
We should also apply the same additions/changes to `CAG` (in `cag.js`) as we do here.

Could be `isCAG` instead of `isCAF`, but then everywhere I think users might have to check `isCAF || isCAG`. That might get annoying.

Maybe instead, we could change from `isCAF` to something generic like `isCancelable` (or something easier to type and not misspell!), and then have another flag that distinguishes CAF vs CAG.

Thoughts?